webapp/components/generic/SearchableInput/SearchableInput.vue

```diff
@@ -0,0 +1,236 @@
+<template>
+  <div
+    class="searchable-input"
+    aria-label="search"
+    role="search"
+    :class="{
+      'is-active': isActive,
+      'is-open': isOpen,
+    }"
+  >
+    <div class="field">
+      <div class="control">
```

Comment:
That's a lot of wrappers! One should be enough to position the `ds-button` and `ds-select`, no? :wink:

Reply:
I would even prefer this to be an `<input>` rather than a `<div>` but since our beloved styleguide adds a bunch of its own wrapper `divs` it would probably mess things up... :laughing: